snmp_types.yaml

```diff
@@ -0,0 +1,87 @@
+tosca_definitions_version: cloudify_dsl_1_0
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.2m4/types.yaml
+  - http://www.getcloudify.org/spec/diamond-plugin/1.2m4/plugin.yaml
+  - http://www.getcloudify.org/spec/openstack-plugin/1.2m4/plugin.yaml
+
+
+node_types:
+  SNMPSecurityGroup:
```

Comment:
The SNMPSecurityGroup should not be part of the snmp-types.yaml as it is very much openstack specific. Even if it will be duplicated in the 2 main blueprints, it should only be configured there inlined into node_templates